boards/arm/96b_argonkey/96b_argonkey.yaml

```diff
@@ -11,5 +11,6 @@ flash: 1024
 supported:
   - gpio
   - i2c
+  - hts221
```

Comment:
You can add vl53l0x as well, right ?

Reply:
Correct!
And LPS22HB and LSM6DSL.

Reply:
Figured they will get added as we convert / test them.